pkg/probe/log_fortianalyzer.go

```diff
@@ -0,0 +1,48 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaResults struct {
+	Registration string  `json:"registration"`
+	Connection   string  `json:"connection"`
+	Received     float64 `json:"received"`
+}
+
+type LogAna struct {
+	Results LogAnaResults `json:"results"`
+	VDOM    string        `json:"vdom"`
+}
+
+func probeLogAnalyzer(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		logAnaInfo = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_registration",
```

Comment:
```suggestion
			"fortigate_log_fortianalyzer_registration_info",
```
Info metrics have the `_info` suffix. Sry, I have forgot to add it myself in the last suggestion